types/fh-sync.d.ts

```diff
@@ -98,22 +98,22 @@ declare module 'fh-sync' {
     function handleCollision(dataset_id: string, onCollision: (dataset_id: string, hash: string, timestamp: any, uid: string, pre: any, post: any, meta_data: any, callback: StandardCb<any>) => void): void;
     function globalHandleCollision(onCollision: (dataset_id: string, hash: string, timestamp: Date, uid: string, pre: any, post: any, meta_data: any, callback: StandardCb<any>) => void): void;
 
-    function listCollisions(dataset_id: string, onList: (dataset_id: string, callback: StandardCb<{ [hash: string]: any }>, meta_data: any, callback: StandardCb<any>) => void): void;
-    function globalListCollisions(onList: (dataset_id: string, callback: StandardCb<{ [hash: string]: any }>, meta_data: any, callback: StandardCb<any>) => void): void;
+    function listCollisions(dataset_id: string, onList: (dataset_id: string, meta_data: any, callback: StandardCb<{ [hash: string]: any }>) => void): void;
+    function globalListCollisions(onList: (dataset_id: string, meta_data: any, callback: StandardCb<{ [hash: string]: any }>) => void): void;
 
     function removeCollision(dataset_id: string, onRemove: (dataset_id: string, collision_hash: string, meta_data: any, callback: StandardCb<any>) => void): void;
 
     function interceptRequest(dataset_id: string, onIntercept: (dataset_id: string, interceptor_params: SyncInterceptParams, callback: NoRespCb) => void): void;
     function interceptResponse(): void;
 
-    function setConfig(config:any): void;
-    
+    function setConfig(config: any): void;
+
     function globalInterceptRequest(onIntercept: (dataset_id: string, interceptor_params: SyncInterceptParams, callback: NoRespCb) => void): void;
     function globalInterceptResponse(): void;
-    
-    function setGlobalHashFn((object: any)=> any): void;
-    function setRecordHashFn((object: any)=> any): void;
-    
+
+    function setGlobalHashFn(hashFunction: (target: any) => string): void;
+    function setRecordHashFn(hashFunction: (object: any) => any): void;
```

Comment:
I think `setGlobalHashFn` and `setRecordHashFn` have the signature `(datasetId: string, hashFunction: (target: any) => string)`?